source/tests/CMakeLists.txt

```diff
@@ -1,13 +1,33 @@
 
+#
+# Configure CMake own project
+#
+
+cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../cmake")
+
+include(${CMAKE_CURRENT_SOURCE_DIR}/../../cmake/Custom.cmake)
```

Comment:
This will not work if the test directory is copied to some other place (which was the intention, I think?)